app/build.gradle

```diff
@@ -209,6 +214,9 @@ dependencies {
     }
 
     androidTestImplementation "org.jetbrains.kotlin:kotlin-reflect:${kotlin_version}"
+    compileOnly files('libs/amazon-device-messaging-1.0.1.jar')
```

Comment:
a-c already includes this dependency which means there's a big chance this could go out of sync. Can you file a follow-up bug?

Reply:
> a-c already includes this dependency which means there's a big chance this could go out of sync.

We use `compileOnly` as well so we build against the same JAR, but don't include it either. I've filed https://github.com/mozilla-mobile/android-components/issues/4157 so that we expose the SDK we compile against to make it easier for consuming apps to do the same.